nautobot/extras/views.py

```diff
@@ -707,7 +707,7 @@ def post(self, request, class_path):
                 job.class_path,
                 job_content_type,
                 request.user,
-                data=form.cleaned_data,
+                data=form.get_serializeable_data(),
```

Comment:
With `form.get_serializeable_data()` moved to `job.get_serializeable_data()`, this could be replaced with:
```suggestion
                data=job.get_serializeable_data(form.cleaned_data),
```